docs/modules/getting-started/pages/index.adoc

```diff
@@ -1,3 +1,52 @@
-= Getting Started
+= Getting Started with Mequal
 
-include::ROOT:partial$wip.adoc[]
\ No newline at end of file
+== 1. Quickstart with a Container
+
+include::getting-started:partial$quickstart.adoc[]
+
+== 2. Build and Run from Source Code
+
+=== a. Build and run inside a container (Recommended)
+
+We can easily start editing the source code and maybe even adding some policies of our own without worrying about the tooling underneath.
+
+[source,bash]
+----
+$ git clone https://github.com/project-ncl/mequal.git
+$ cd mequal
+$ podman build -t "mequal-testing" -f ./container_files/Containerfile .
+$ cat input/good/sbom-example.json | podman run -i --rm mequal-testing
+----
+
+[TIP]
+====
+The https://google.com[./container_files/image-run.sh] script can be edited to decide what should be ran inside the container. Can decide to do evaluations using Conforma or using the OPA client embedded within it, running unit tests, change evaluation parameters, etc.
```

Comment:
Is the url correct?

Reply:
Yeah it's just a placeholder until we move the code, I just use google out of habit

Reply:
Ah. I would add an attribute: https://docs.asciidoctor.org/asciidoc/latest/attributes/document-attributes/ to which would point to the Mequal repo for now. It will be broken, but better, I think. Plus, you will control the URL in a single place.

This can be added to https://github.com/project-ncl/mequal/blob/main/antora-playbook.yml globally as well.

Reply:
Sure, that makes more sense